### PR TITLE
refactor(PaginatedItemsSummary): FRWM-696-convert to Quark Filter

### PR DESCRIPTION
**Pull request by 100stacks**
JIRA: https://jira.rax.io/browse/FRMW-696

- [x] Dev LGTM
- [x] Dev LGTM

---
Refactor `rxPaginate.PaginatedItemsSummary` to Quark Filter.

### CODE REVIEW

**Comment by 100stacks** [comment]
## `PaginatedItemsSummary` Quark Filter

### Demo Page
screen shot 2016-02-15 at 10 51 08 am


### API Docs
screen shot 2016-02-15 at 10 21 23 am

**Comment by CITguy** [comment]
LGTM

File: src/quarks/PaginatedItemsSummary/docs/PaginatedItemsSummary.docs.html [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<p>
+  Provides a formatted string with the current range of items in total items, for example, 26-50 of 500 items.
```
Comment (parlarjb) [inline src/quarks/PaginatedItemsSummary/docs/PaginatedItemsSummary.docs.html]: Could you please put quotes around the example, i.e. `for example "26-50 of 500 items"`, to clearly separate what this is doing?

**Comment by 100stacks** [comment]
@parlarjb, feedback incorporated. :+1:

**Comment by parlarjb** [comment]
LGTM

**Pull request merged**